### ISSUE TITLE
badges meta tracking issue

### ISSUE THREAD

**Issue by nikhiljha** [ocf/projects#50]
There's a lot of work to be done to make badges a reality, and some of it is non-technical work. If you're here because this is a good first issue, maybe try drawing some badge art for the badge ideas in [ocf/badgr](https://github.com/ocf/badgr/issues) or at the link at the bottom of this issue.

## Tasks
- [ ] come up with 20-25 badges to implement
- [ ] finish the per-badge tasks for each badge
- [ ] dump github/jenkins/etc data into a message bus
- [ ] write a listener that consumes the message bus and issues badges only once, just a script that calls https://pypi.org/project/badgr-lite/ is good enough
- [ ] figure out how to setup SAML

## Per-Badge Tasks (separate issue for each badge)
- Come up with a creative title for the badge.
- Come up with a creative short description for the badge.
- Write a short "how to get this badge" blurb.
- Create `.svg` or `.png` artwork for the badge.

## External Documents
Badge Ideas: https://notes.ocf.berkeley.edu/ICLWok3jQYSjslTfxb-Evw#

**Comment by nikhiljha** [comment]
New information that was discussed in IRC at some point:

- use https://pypi.org/project/badgrclient/ instead of badgr-lite
